README.md

```diff
@@ -43,65 +43,297 @@
+
+### Rust Crate
+
+For using `aquadoggo` in your Rust project, you can add it as a dependency with the following command:
+
+```bash
+cargo add aquadoggo
 ```
 
-You can also run the node server simply as a [command line application][`command line application`]:
+## Usage
+
+### Run node
+
+You can also run the node simply as a [command line application](/aquadoggo_cli). `aquadoggo` can be configured in countless ways for your needs, read our [configuration](/aquadoggo_cli/README.md#Usage) section for more examples, usecases and an overview of configuration options.
 
 ```bash
-# Compile and run local node at http://localhost:2020/graphql
-cargo run
+# Start a local node on your machine, go to http://localhost:2020/graphql for using the GraphQL playground
+aquadoggo
+
+# Check out all configuration options
+aquadoggo --help
 
 # Enable logging
-RUST_LOG=aquadoggo=info cargo run
+aquadoggo --log-level info
 ```
 
-.. or run it inside a [Docker](https://hub.docker.com/r/p2panda/aquadoggo) container:
+### Docker
+
+For server deployments you might prefer using [Docker](https://hub.docker.com/r/p2panda/aquadoggo) to run `aquadoggo`.
 
 ```bash
-docker run -p 2020:2020 -e RUST_LOG=aquadoggo=info p2panda/aquadoggo
+docker run -p 2020:2020 -p 2022:2022 -e LOG_LEVEL=info p2panda/aquadoggo
 ```
 
-[`Tauri`]: https://tauri.studio
+### Embed node
 
-## Installation
+Run the node directly next to the frontend you're building for full peer-to-peer applications by using the [`aquadoggo`](/aquadoggo) Rust crate. Check out our [Tauri](https://github.com/p2panda/tauri-example) example for writing a desktop app.
 
-```sh
-cargo add aquadoggo
+```rust
+use aquadoggo::{Configuration, Node};
+use p2panda_rs::identity::KeyPair;
+
+let config = Configuration::default();
+let key_pair = KeyPair::new();
+let node = Node::start(key_pair, config).await;
 ```
 
+### FFI bindings
+
+If you are not working with Rust you can create FFI bindings from the `aquadoggo` crate into your preferred programming language. Dealing with FFI bindings can be a bit cumbersome and we do not have much prepared for you (yet), but check out our [Meli](https://github.com/p2panda/meli/) Android project as an example on how we dealt with FFI bindings for Dart / Flutter.
+
+## Resources
+
+- 🐬 Deploy your own `aquadoggo` following the [tutorial](https://p2panda.org/tutorials/aquadoggo)
+- 🛠️ Create your own schemas using [`fishy`](https://github.com/p2panda/fishy)
+- 🛼 Open the GraphQL playground in your browser, served under `http://localhost:2020/graphql`
+- 📖 Try the [mushroom app tutorial](https://p2panda.org/tutorials/mushroom-app)
+- 🔬 Manually publish JSON encoded data to a node [`send-to-node`](https://github.com/p2panda/send-to-node)
+- 🐼 [Learn more](https://p2panda.org/learn/) about how p2panda works
```

Comment:
Even though it's true it's confusing to mention JSON here maybe, people might think that p2panda data in general is JSON encoded